.circleci/config.yml

```diff
@@ -136,7 +136,7 @@ jobs:
       - bazel_install
       - bazel_add_rbe_credential
       - bazel:
-          command: bazel build //:distribution
+          command: bazel build //:distribution-zip-mac
```

Comment:
should depend on `distribution-targz` because this test is ran on a linux machine